Lots of "failed to delete the execution xxx" logs when many executions runs in the same time

This is caused by the execution sweeper and doesn't effect the functionality but it's better to fix it.
```
Feb  3 09:14:00 172.28.0.1 core[2219]: 2021-02-03T09:14:00Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 527: task 3557 not found
Feb  3 09:14:00 172.28.0.1 core[2219]: 2021-02-03T09:14:00Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 527: task 3556 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 527: task 3558 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 527: task 3559 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 527: execution 527 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 526: task 3531 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 529: task 3554 not found
Feb  3 09:14:01 172.28.0.1 core[2219]: 2021-02-03T09:14:01Z [ERROR] [/pkg/task/execution.go:170]: failed to delete the execution 529: task 3553 not found
```

--- a/src/pkg/task/execution.go
+++ b/src/pkg/task/execution.go
@@ -167,8 +167,11 @@ func (e *executionManager) sweep(ctx context.Context, vendorType string, vendorI
 				continue
 			}
 			if err = e.Delete(ctx, execution.ID); err != nil {
+				// the execution may be deleted by the other sweep operation, ignore the not found error
+				if errors.IsNotFoundErr(err) {
+					continue
+				}
 				log.Errorf("failed to delete the execution %d: %v", execution.ID, err)
-				continue
 			}
 		}
 	}
@@ -314,6 +317,11 @@ func (e *executionManager) Delete(ctx context.Context, id int64) error {
 				WithMessage("the execution %d has tasks that aren't in final status, stop the tasks first", id)
 		}
 		if err = e.taskDAO.Delete(ctx, task.ID); err != nil {
+			// the tasks may be deleted by the other execution deletion operation in the same time(e.g. execution sweeper),
+			// ignore the not found error for the tasks
+			if errors.IsNotFoundErr(err) {
+				continue
+			}
 			return err
 		}
 	}

